common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -109,16 +109,7 @@ function(HTML5Video, Resizer) {
         // starts playing. Just after that configurations can be applied.
         state.videoPlayer.ready = _.once(function() {
             if (!state.isFlashMode() && state.speed != '1.0') {
-                // Work around a bug in the Youtube API that causes videos to
-                // play at normal speed rather than at the configured speed in
-                // Safari.  Setting the playback rate to 1.0 *after* playing
-                // started and then to the actual value tricks the player into
-                // picking up the speed setting.
-                if (state.browserIsSafari && state.isYoutubeType()) {
```

Comment:
@karandikar I think we can remove [these lines](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/video/03_video_player.js#L151-L157) safely and then we can remove these variables from tests too if it makes good sense.